frontend/config/dev.webpack.config.js

```diff
@@ -85,8 +88,8 @@ const webpackProxy = {
       ws: true,
       pathRewrite: { '^/api/k8s': '' },
     },
-    pluginProxy('hac-dev'),
-    pluginProxy('hac-infra'),
+    process.env.LOCAL_HAC_DEV ? pluginProxy('hac-dev') : {},
```

Comment:
@karelhala we may need a PSA in the core channel that this change will impact devs trying to run RHTAP locally. Alternatively we could update our `start:k8s` script to include HAC_DEV at a minimum? Either way is fine with me.

Reply:
Good point, I've updated the `start:proxy:beta` to include this new env var.

Reply:
The env vars passing trough is kinda wonky and you would have to go to package.json and directly romove it in order to make it working. I've rather added entry in the documentation about this flag.